tests/testthat/test-param_set.R

```diff
@@ -7,18 +7,24 @@ source(test_path("../helper-objects.R"))
 # ------------------------------------------------------------------------------
 
 test_that('recipe with no steps', {
+  withr::local_options(lifecycle_verbosity = "quiet")
+
   bare_info <- dials::parameters(bare_rec)
   check_param_set_tibble(bare_info)
   expect_equal(nrow(bare_info), 0)
```

Comment:
Since the methods are already tested in their respective packages, I would remove all of these tests.

If you'd like, you could keep 1 simple test per method in this package, just to ensure that calling `parameters()` "still works" for now - but it should be clear in the testthat message that it is deprecated

Something like:
https://github.com/r-lib/vctrs/blob/main/tests/testthat/test-lifecycle-deprecated.R

`local_lifecycle_silence()` is just https://github.com/r-lib/vctrs/blob/e86bb9a4b57459eb56099c0b52af37877a9bf515/R/compat-lifecycle.R#L149

Reply:
Regarding`local_lifecycle_silence()`: do you mean we should use something with `rlang::local_options()` here instead of a function from `withr`? Or is the second link just for illustration?

Reply:
just for illustration. you can use either withr or rlang here for `local_options()`, they are basically equivalent